src/Tools/dotnet-monitor/DataAnnotationValidateOptions.cs

```diff
@@ -7,7 +7,11 @@
 using System.Collections.ObjectModel;
 using System.ComponentModel.DataAnnotations;
 
+#if EXTENSION
+namespace Microsoft.Diagnostics.Monitoring.Extension.Common
```

Comment:
I'm assuming this is temporary, given our future plans to physically separate dotnet monitor from the extensions?

Reply:
Yes, linking in files is good for now to avoid code duplication, especially if we don't intend to ship libraries. If there is a bifurcation, then whoever does it needs to just copy the file as-is.